Add files via upload

kittytheastronaut opened the pull request:
Adding the analysis for the CCW torque dependences on rotator angle for the ComCam campaign data

kittytheastronaut committed:
Add files via upload

Adding the analysis for the CCW torque dependences on rotator angle for the ComCam campaign data

Signed-off-by: Marina Pavlovic <marinap@mi.sanu.ac.rs>

b1quint reviewed (changes requested):
Could you move this file to the `notebooks/tel_and_site/subsys_req_ver/tma/` folder? In addition, may I ask you to replace the `:` in the filename with `_`?

kittytheastronaut committed:
Rename SITCOM-1853_CCW:Rotator_ComCam.ipynb to SITCOM-1853_CCW_Rotator_ComCam.ipynb

Name of the file changed from: 

SITCOM-1853_CCW:Rotator_ComCam.ipynb to SITCOM-1853_CCW_Rotator_ComCam.ipynb to: 

Rename SITCOM-1853_CCW_Rotator_ComCam.ipynb to SITCOM-1853_CCW_Rotator_ComCam.ipynb

Signed-off-by: Marina Pavlovic <marinap@mi.sanu.ac.rs>

kittytheastronaut committed:
Move SITCOM-1853_CCW_Rotator_ComCam.ipynb to notebooks/tel_and_site/subsys_req_ver/tma/SITCOM-1853_CCW_Rotator_ComCam.ipynb

The notebook is moved to another directory: notebooks/tel_and_site/subsys_req_ver/tma/

Signed-off-by: Marina Pavlovic <marinap@mi.sanu.ac.rs>

kittytheastronaut commented:
Notebook is moved to notebooks/tel_and_site/subsys_req_ver/tma/, and the name is changed so it doesn't have ":" now. Let me know if there is anything else that I can do. Thank you!

b1quint reviewed (approved):
Everything seems good now. Thank you!
